terraform/environments/core-network-services/secrets.tf

```diff
@@ -21,6 +21,99 @@ data "aws_secretsmanager_secret_version" "pagerduty_integration_keys" {
   secret_id = data.aws_secretsmanager_secret.pagerduty_integration_keys.id
 }
 
+
+# Data for Firehose Endpoint URL & Key that are held in secrets manager.
+
+# 1. Secrets for Firewall-related log data.
```

Comment:
Could these secrets be refactored to use a for_each declaration? You might need to think about how to supply it with values, but it would make for cleaner code.